stable/defaultstorageclass/templates/deployment.yaml

```diff
@@ -0,0 +1,62 @@
+# This file is generated from `update.sh`. DO NOT EDIT.
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  labels:
+    control-plane: controller-manager
+  name: dstorageclass-controller-manager
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      control-plane: controller-manager
+  strategy: {}
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        control-plane: controller-manager
+    spec:
+      containers:
+      - args:
+        - --secure-listen-address=0.0.0.0:8443
+        - --upstream=http://127.0.0.1:8080/
+        - --logtostderr=true
+        - --v=10
+        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.4.0
+        name: kube-rbac-proxy
+        ports:
+        - containerPort: 8443
+          name: https
+        resources: {}
+      - args:
+        - --metrics-addr=127.0.0.1:8080
+        - --enable-leader-election
+        command:
+        - /manager
+        image: mesosphere/default-storage-class:latest
+        name: manager
+        ports:
+        - containerPort: 443
+          name: webhook-server
+          protocol: TCP
+        resources:
```

Comment:
this should not be hardcoded

Reply:
it's generated from kubebuilder and copied into here from the kubebuilder project.

Reply:
cool, totally invalid

Reply:
autogenerated code can be changed

Reply:
It's standard practice for helm charts to have resources configurable via values as @hectorj2f says. Let's follow this convention please.

Reply:
ok, it's changed now.